telegraf/rootfs/config.toml.tpl

```diff
@@ -1,99 +1,103 @@
-# Set Tag Configuration
-[tags]
+{{ if .Env.GLOBAL_TAGS }}
+[global_tags]
+    {{ range $index, $element := split .Env.GLOBAL_TAGS "," }}
+      {{ $value := split $element ":" }}{{index $value 0}}="{{index $value 1}}"
+    {{end}}
+{{ end }}
```

Comment:
you could have accomplished this same thing with envtpl which includes [sprig](https://github.com/Masterminds/sprig)